src/ui/libs/DatalensChartkit/ChartKit/plugins/Metric/renderer/MetricTile.tsx

```diff
@@ -115,9 +115,9 @@ export class MetricTile extends React.PureComponent<MetricTileProps> {
         }
 
         const {config, callback} = getGraph(chartConfig, chart, undefined, isMobile);
-
         return (
             <HighchartsReact
+                key={Math.random()}
```

Comment:
We already use such hack [here](https://github.com/gravity-ui/chartkit/blob/main/src/plugins/highcharts/renderer/components/HighchartsComponent.tsx#L162)